holidayextras/brands
# Moving social buttons into globals

#### What does this PR do? (please provide any background)
Moves social media buttons (like twitter and facebook) into globals for everyone to share.

#### How can this be tested?
`cd pattern-library && npm link ../brands`
Run this branch locally with pattern library (npm link as above)
Check everything compiles.

#### Any tech debt?
None

#### Screenshots / Screencast
None

#### What gif best describes how you feel about this work?
![giphy-5](https://cloud.githubusercontent.com/assets/778942/22924204/2c86faee-f29c-11e6-89e4-2287b706e716.gif)



---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---
